### PR TITLE
647 Split repo

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

**Review by Georgi2704 — APPROVED** [review]
works nice !

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[submodule "apps/wfo-ui-surf"]
+	path = apps/wfo-ui-surf
+	url = git@git.ia.surfsara.nl:netdev/automation/projects/orchestrator-ui.git
```
Comment (ricardovdheijden) [inline .gitmodules:3]: I do know this is the only way for now, BUT is this link to a surf-repo something we want to keep in the public facing repo?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
 ```
-npm
-turbo dev
+git clone git@github.com:workfloworchestrator/orchestrator-ui-library.git
+git submodule init
+git submodule update
+cp apps/wfo-ui/.env.example apps/wfo-iu/.env
+# change the values in the env file to point to your orchestrator backend
+# set auth=false or follow the directions bellow this sections
+npm i
```
Comment (ricardovdheijden) [inline README.md:15]: I am getting this error: 
image

File: .gitmodules (hunk 1 above)
Comment (DutchBen) [reply .gitmodules]: No, fixed and to be discussed

**Review by ricardovdheijden — APPROVED** [review]

**Pull request merged**